src/CLIPreTasks/DeprecationChecker/DeprecationChecker.ts

```diff
@@ -24,9 +24,11 @@ export class DeprecationChecker {
       return
     }
 
-    const errMsg = `${chalk.bold(
-      `Your Toolbelt version (${pkgJson.version}) was deprecated`
-    )}. ${Messages.UPDATE_TOOLBELT()}`
+    const errMsg = [
+      `${chalk.bold(`Your Toolbelt version (${pkgJson.version}) was deprecated`)}.`,
+      `To update, you must use the same method you used to install. As the following example(s):`,
+      ...deprecateMessageSwitch(),
```

Comment:
Can you change from `deprecatedMessageSwitch` to `updateFromDeprecatedMessageSwitch` or something like this?